README.md

```diff
@@ -1,36 +1,36 @@
-# strata-bridge-poc
+# Strata Bridge Proof of Concept
 
 [![License: MIT](https://img.shields.io/badge/License-MIT-blue.svg)](https://opensource.org/licenses/MIT)
 [![License: Apache-2.0](https://img.shields.io/badge/License-Apache-blue.svg)](https://opensource.org/licenses/apache-2-0)
 [![ci](https://github.com/alpenlabs/strata-bridge-poc/actions/workflows/lint.yml/badge.svg?event=push)](https://github.com/alpenlabs/strata-bridge-poc/actions)
 
-PoC Implementation of the Strata Bridge.
+A research implementation of Strata Bridge by your friends at [Alpen Labs](https://www.alpenlabs.io/).
 
 > [!IMPORTANT]
-> Majority of this code was written in a two-week period in a hackathon spirit.
-> This is pre-alpha software that is _not_ meant to be used in a production environment.
+> The majority of this code was written in a two-week period in a hackathon spirit.
+> This software was written as a research proof of concept, and is _not_ meant to be used in a production environment!
 
 ## Transaction Graph
 
-The following is the transaction graph that has been implemented in this repo.
+The following is the transaction graph that has been implemented in this repository:
 
 <figure>
-    <img src="./assets/poc-tx-graph.jpg" alt = "poc tx graph" />
-    <figcaption>The Transaction Graph for this PoC focussing on the non-optimistic path.</figcaption>
+    <img src="./assets/poc-tx-graph.jpg" alt = "Transaction graph" />
+    <figcaption>The transaction graph for this repository, focusing on the non-optimistic path.</figcaption>
```

Comment:
```suggestion
    <figcaption>The transaction graph for this repository, focussing on the non-optimistic path.</figcaption>
```

Reply:
Ah, I wasn't aware this was a (presumably non-American English) spelling!

Reply:
I wasn't aware that's the American spelling. 😅

Fine to leave as is then.

Reply:
I have no preference. Apparently both spellings are generally considered valid. Already reverted.